index.html

```diff
@@ -0,0 +1,195 @@
+                    <h3 class="events">Keynotes</h3>
+                    <p class="event-text">
+                        Hear from thought leaders in education develop new skills and find new opportunities.
+                    </p>
+                </div>
+                <div class="single-event">
+                    <svg xmlns="http://www.w3.org/2000/svg" width="50" height="50" fill="currentColor" class="bi bi-briefcase-fill x-1" viewBox="0 0 16 16">
+                        <path d="M6.5 1A1.5 1.5 0 0 0 5 2.5V3H1.5A1.5 1.5 0 0 0 0 4.5v1.384l7.614 2.03a1.5 1.5 0 0 0 .772 0L16 5.884V4.5A1.5 1.5 0 0 0 14.5 3H11v-.5A1.5 1.5 0 0 0 9.5 1h-3zm0 1h3a.5.5 0 0 1 .5.5V3H6v-.5a.5.5 0 0 1 .5-.5z"/>
+                        <path d="M0 12.5A1.5 1.5 0 0 0 1.5 14h13a1.5 1.5 0 0 0 1.5-1.5V6.85L8.129 8.947a.5.5 0 0 1-.258 0L0 6.85v5.65z"/>                    </svg>
+                    <h3 class="events">Networks</h3>
+                    <p class="event-text">
+                        An unsurpassed opportunity to meet fellow educators, teachers, leaders and officials.
+                    </p>
+                </div>
+                <div class="single-event">
+                    <svg xmlns="http://www.w3.org/2000/svg" width="50" height="50" fill="currentColor" class="bi bi-boxes x-1" viewBox="0 0 16 16">
+                        <path d="M7.752.066a.5.5 0 0 1 .496 0l3.75 2.143a.5.5 0 0 1 .252.434v3.995l3.498 2A.5.5 0 0 1 16 9.07v4.286a.5.5 0 0 1-.252.434l-3.75 2.143a.5.5 0 0 1-.496 0l-3.502-2-3.502 2.001a.5.5 0 0 1-.496 0l-3.75-2.143A.5.5 0 0 1 0 13.357V9.071a.5.5 0 0 1 .252-.434L3.75 6.638V2.643a.5.5 0 0 1 .252-.434L7.752.066ZM4.25 7.504 1.508 9.071l2.742 1.567 2.742-1.567L4.25 7.504ZM7.5 9.933l-2.75 1.571v3.134l2.75-1.571V9.933Zm1 3.134 2.75 1.571v-3.134L8.5 9.933v3.134Zm.508-3.996 2.742 1.567 2.742-1.567-2.742-1.567-2.742 1.567Zm2.242-2.433V3.504L8.5 5.076V8.21l2.75-1.572ZM7.5 8.21V5.076L4.75 3.504v3.134L7.5 8.21ZM5.258 2.643 8 4.21l2.742-1.567L8 1.076 5.258 2.643ZM15 9.933l-2.75 1.571v3.134L15 13.067V9.933ZM3.75 14.638v-3.134L1 9.933v3.134l2.75 1.571Z"/>
+                    </svg>
+                    <h3 class="events">Exhibition</h3>
+                    <p class="event-text">
+                        Alongside the dynamic presenters and sessions on offer during the education summit.
+                    </p>
+                </div>
+            </div>
+            <h3 class="view-program">
+                <a href="#" class="a-text">View Full Program</a>
+            </h3>
+            <div class="register col-6 mx-auto">
+                <button class="reg-btn display-5 my-5 p-3">Register Yourselves</button>
+            </div>
+        </section>
+        <!-- Speaker section goes there -->
+        <section>
+            <h2 class="text-center mt-5">Featured Speakers</h2>
+            <div class="br-line"></div>
+                <article class="speaker-x mt-5" id="speakers">
+                </article>
+
+        </section>
```

Comment:
- [ ] Please add the "More" button to the end of the "Speakers" section on the mobile view to match the given template.

![image](https://user-images.githubusercontent.com/62290545/225335338-78540c89-30a2-47ce-bcfb-0808d74df50a.png)

Reply:
Definitely, thank you so much pointing that out